### ISSUE TITLE
i have an error in the activation in panda gamepad apk

### ISSUE THREAD

**Issue by cristicchi** [dysquard/pgpa#17]
Hi, I bought the application from the playstore but I have the following error screen, I hope you can help me thank you very much my device is a Huawei p20 (ANE-LX3) SOFTWARE VERSION 9.1.0.240
![image](https://user-images.githubusercontent.com/54999941/71009280-920e5580-20af-11ea-8c74-104f0ce1e44a.png)
if the application is open in my device

**Comment by dysquard** [comment]
did u try as it said?

**Comment by eli5555** [comment]
i have the same problem, it keep saying make sure phone's usb debugging is on and reconnect your phone to pc via USB, i have done all that and done all the troubleshooting. i even contacted Oppo and they said the issue is not from the phone, i also tried it on mac and pc still the same issue. even tried different USB cables 

<img width="574" alt="Screen Shot 2020-02-12 at 3 27 45 pm" src="https://user-images.githubusercontent.com/60955623/74304364-b84d2a80-4dac-11ea-812e-2da74efc4ac8.png">

**Comment by sirredneck** [comment]
Good luck. I bought the activator that works 100% guaranteed and still
won't work on my LG Phoenix 4
